nilearn/regions/signal_extraction.py

```diff
@@ -15,6 +15,150 @@
 from .. import masking
 from ..image import new_img_like
 
+INF = 1000 * np.finfo(np.float32).eps
+
+
+def _check_shape_affine_label_img(labels_img, target_shape, target_affine):
+    """Validate shapes and affines of labels.
+
+    Parameters
+    ----------
+    labels_img : Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
```

Comment:
this link is now dead: not sure which one to use instead

Reply:
Appears in several other doc strings:

```bash
$  grep -rnw nilearn -e 'nilearn.github.io' 

nilearn/reporting/glm_reporter.py:640:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/reporting/glm_reporter.py:645:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/reporting/glm_reporter.py:722:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/reporting/glm_reporter.py:897:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/reporting/glm_reporter_templates/report_body_template.html:82:    Built using <a href="https://nilearn.github.io">Nilearn</a>.
nilearn/__init__.py:6:http://nilearn.github.io.
nilearn/masking.py:37:        See https://nilearn.github.io/stable/manipulating_images/input_output.html  # noqa:E501
nilearn/masking.py:762:        See https://nilearn.github.io/stable/manipulating_images/input_output.html  # noqa:E501
nilearn/image/resampling.py:3:See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/decoding/decoder.py:85:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/_utils/exceptions.py:56:                "See https://nilearn.github.io/stable/manipulating_images/"
nilearn/_utils/niimg.py:113:        See https://nilearn.github.io/stable/manipulating_images/input_output.html  # noqa:E501
nilearn/_utils/niimg.py:158:        See https://nilearn.github.io/stable/manipulating_images/input_output.html  # noqa:E501
nilearn/decomposition/_base.py:111:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/decomposition/_base.py:333:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/decomposition/_base.py:386:            See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/decomposition/_base.py:479:            See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/decomposition/_base.py:554:            See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/decomposition/_multi_pca.py:115:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/plotting/img_plotting.py:3:See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/plotting/img_plotting.py:1014:        See http://nilearn.github.io/auto_examples/01_plotting/plot_demo_glass_brain_extensive.html
nilearn/plotting/html_surface.py:203:        https://nilearn.github.io/stable/auto_examples/01_plotting/plot_3d_map_to_surface_projection.html
nilearn/version.py:26:    'https://nilearn.github.io/stable/introduction.html#installation')
nilearn/regions/signal_extraction.py:27:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/regions/signal_extraction.py:62:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/regions/signal_extraction.py:108:        See http://nilearn.github.io/manipulating_images/input_output.html
nilearn/regions/signal_extraction.py:119:        See http://nilearn.github.io/manipulating_images/input_output.html
```

Reply:
fixed it

will open an issue to fix the others

Reply:
Actually, there is only a handful of Niimg-like object parameters in the whole code base that have this link.

It is unclear why those should more than any other, so I am removing them.

Reply:
OK actually they should be replaced by  

```
See :ref:`extracting_data`.
```

Reply:
Ah yes that's right, I replaced them a while ago but must have missed some. Thx for catching them!